crates/driver/src/infra/persistence/dto.rs

```diff
@@ -0,0 +1,398 @@
+use {
```

Comment:
We had two options here:
- create the dto for storing this information in the S3
- using the already existing `Auction` and `Liquidity` structs and derive everything needed for serialization, etc

I decided to do the first option mostly for better data control and encapsulation. But I am open to changes, of course.

Reply:
I think given that @fhenneke's goal is to use this file and replay it against an existing solver implementation (Quasimodo) using the exact same representation as what we send to solvers would make sense. In fact, an alternative implementation approach would be to simply write another solver engine that doesn't compute anything but simply updates the received instance file and returns an empty solution.